designsafe/static/scripts/ng-designsafe/services/project-service.js

```diff
@@ -105,6 +105,43 @@ export class ProjectService {
         return from(piPromise);
     }
 
+    getPiData(piMap) {
+        const obs = [piMap.pi, ...piMap.coPis].map(u =>
```

Comment:
This can cause errors if `piMap.coPis` is undefined, which seems to be the case in some projects (I'm looking at PRJ-2789 now).